db/migrate/20200709080259_remove_image_from_lot.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveImageFromLot < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :lots, :image
```

Comment:
```suggestion
    remove_column :lots, :image, :string
```
So it would be reversible